packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -320,10 +330,56 @@ markdown based on [CommonMark](http://commonmark.org/) flavor:
 -   Italics - `*Italics*`
 -   Numbered list - `1. Numbered list`
 -   Bulleted list - `* Bulleted list`
+-   Absolute URL links - `<Absolute URI link>` (For more details on the markdown syntax for absolute URL links, see [Autolinks in CommonMark](https://spec.commonmark.org/0.30/#autolink))
+
+_Configurations on Tiptap to support only absolute links_:
+
+Install the [link extension](https://tiptap.dev/api/marks/link) mark from Tiptap and initialize the `Links` with the following configurations:
+
+1.  Set regular expression in [validate](https://tiptap.dev/api/marks/link#validate) field to support only `HTTP` and `HTTPS` absolute links in the editor.
+2.  Set [openOnClick](https://tiptap.dev/api/marks/link#open-on-click) to `false` for the editor, to restrict the user from opening a link from the editor by clicking. Users can open the link only by
+    `Right-click >> Open link in new tab` from the editor.
+3.  Set [autoLink](https://tiptap.dev/api/marks/link#autolink) to `true`, to add the valid link automatically when typing.
+4.  Set [linkOnPaste](https://tiptap.dev/api/marks/link#link-on-paste) to `false` which will attach the URL to current selected text in the editor, converting it into a hyperlink. If it is `true`,
+    pasting a link to the selection will add the link behind the word which is not supported for the initial pass.
 
 The `nimble-rich-text-viewer` will be responsible for converting the input markdown string to HTML Fragments with the help of
 `prosemirror-markdown` parser, which is then converted to HTML string and rendered into the component to view all rich text content.
 
+_Implementation details for supporting absolute link:_
```

Comment:
We should have symmetry in validation both ways.
So if tiptap validates links are http and http for the rich-text-editor,
how should the rich-text-viewer behave for links that fail validation?
First reaction is not treat them as links, they just become text. I assume that is how the editor side will behave for unsupported schemes, just treat them as text

Reply:
@rajsite, yes, currently the `rich-text-viewer` does not have any validation to consider `http` / `https`. 

But as you mentioned, we will update the `rich-text-viewer` to render the links as plain text, if the links do not have a `http` / `https` scheme. And that is how the `rich-text-editor` will also behave.

Reply:
Prototyped the feasibility of validation for HTTP/HTTPS scheme in `rich-text-viewer`'s markdown parsing logic (Prose Mirror/markdown -it) and updated the spec document to mention about the possibility of symmetrical validation in both viewer and editor.